core/storage/question/gae_models_test.py

```diff
@@ -492,6 +542,38 @@ def test_get_question_skill_links_equidistributed_by_skill(
         question_ids = [link.question_id for link in question_skill_links]
         self.assertEqual(question_ids.count('question_id2'), 1)
 
+    def test_get_random_question_skill_links_equidistributed_by_skill(self):
+        questionskilllink_model1 = (
+            question_models.QuestionSkillLinkModel.create(
+                'question_id1', 'skill_id1', 0.1)
+            )
+        questionskilllink_model2 = (
+            question_models.QuestionSkillLinkModel.create(
+                'question_id2', 'skill_id1', 0.5)
+            )
+        questionskilllink_model3 = (
+            question_models.QuestionSkillLinkModel.create(
+                'question_id3', 'skill_id2', 0.8)
+            )
+        questionskilllink_model4 = (
+            question_models.QuestionSkillLinkModel.create(
+                'question_id4', 'skill_id2', 0.9)
+            )
+        question_models.QuestionSkillLinkModel.put_multi_question_skill_links(
+            [questionskilllink_model1, questionskilllink_model2,
+             questionskilllink_model3, questionskilllink_model4])
+        question_skill_links = (
+            question_models.QuestionSkillLinkModel.
+            get_question_skill_links_equidistributed_by_skill(
+                2, ['skill_id1', 'skill_id2']
+            )
+        )
+        self.assertEqual(len(question_skill_links), 2)
+        self.assertTrue(questionskilllink_model1 in question_skill_links or
+                        questionskilllink_model2 in question_skill_links)
+        self.assertTrue(questionskilllink_model3 in question_skill_links or
+                        questionskilllink_model4 in question_skill_links)
```

Comment:
See above comments about testing random functions and change the test

Reply:
Done.